packages/devtools_app/lib/src/example/conditional_screen.dart

```diff
@@ -82,10 +282,50 @@ class _ExampleConditionalScreenBodyState
             color: Colors.grey[50],
             child: const CenteredCircularProgressIndicator(),
           ),
+        evaluator,
       ],
     );
   }
 
+  void highlightDropdown(bool directionDown) {
```

Comment:
nit: this method seems like functionality that could be built into the auto-complete behavior. I don't see anything different about this case than a default implementation.

Reply:
Good point, moved to built in default behavior.